app/jobs/api/issues_open_issue_job.rb

```diff
@@ -6,6 +6,7 @@ class IssuesOpenIssueJob < IssueEventJob
 
     def payload(params)
       {
+        pull_request: params[:pull_request],
```

Comment:
Oh, I assume in discovery you found that the webhook doesn't carry a PR object ?